READMEs/data-nfts-and-datatokens-flow.md

```diff
@@ -39,6 +39,15 @@ pip3 install --pre ocean-lib
 
 - Mac users: if you encounter an "Unsupported Architecture" issue, then install including ARCHFLAGS: `ARCHFLAGS="-arch x86_64" pip install ocean-lib`. [[Details](https://github.com/oceanprotocol/ocean.py/issues/486).]
 
+- in order to avoid errors from `brownie`, such as `transaction underpriced`,
+we recommend setting the priority fee before interacting with the smart contracts
+in the python console like this:
+    ```python
+    import brownie.network
+    from brownie.network import priority_fee
+    
+    priority_fee(brownie.network.chain.priority_fee)
+    ```
```

Comment:
This is a start. However it has issues:
- It's not a problem in ganache, only the remote flow. So whatever you put in, it should be in simple-remote.md README, not the simple ganache one.
- doesn't the current fix simply just set the value to what it was already set to? We would need to set it to something different. And, I'm looking at the brownie docs, there are _many_ potential different cases to cover.
- it's too verbose
- it's too early in the README - we haven't even asked people to open the console yet. 

But I have a simple fix: just point to the brownie docs, in the right place!

What to do, specifically: 
- remove the current change
- in simple-remote.md, in section "Setup in Python", at the very end, put in something like the following.

"""
If you get a gas-related error like "transaction underpriced", you'll need to change the `priority_fee` or `max_fee`. See details in [brownie docs](https://eth-brownie.readthedocs.io/en/stable/core-gas.html).
"""

That's it! :)

Reply:
That fix helped for our polygon test, because apparently I have set before with different values for the priority fee and it failed randomly. After I have used the recommended fee for polygon which will change automatically. 
I will change it as you suggested above.